strawberryfields/backends/fockbackend/ops.py

```diff
@@ -328,20 +320,17 @@ def phase(theta, trunc):
 
 
 @functools.lru_cache()
-def beamsplitter(t, r, phi, trunc):
-    r""" The beamsplitter :math:`B(cos^{-1} t, phi)`.
+def beamsplitter(theta, phi, trunc):
+    r""" The beamsplitter :math:`B(\theta, \phi)`.
 
-    Uses the `BSgate operation from The Walrus`_ to calculate the beamsplitter.
+    Uses the `beamsplitter operation from The Walrus`_ to calculate the beamsplitter.
 
-    .. _`BSgate operation from The Walrus`: https://the-walrus.readthedocs.io/en/latest/code/api/thewalrus.fock_gradients.BSgate.html
+    .. _`beamsplitter operation from The Walrus`: https://the-walrus.readthedocs.io/en/latest/code/api/thewalrus.fock_gradients.beamsplitter.html
     """
     # pylint: disable=bad-whitespace
 
-    theta = np.arccos(t)
-    BS_tw, _, _ = BSgate(theta, phi, cutoff=trunc)
-
     # TODO: Transpose needed because of different conventions in SF and The Walrus. Remove when The Walrus is updated.
```

Comment:
Would the PR in The Walrus have an impact on this TODO comment?

Reply:
I'll look into it. The thing is that in TW the convention for the 2-mode gate indices is out1-out2-in1-in2, while in SF it's out1-in1-out2-in2